src/components/Tables/DomainTable/DomainTable.tsx

```diff
@@ -21,6 +29,7 @@ import styles from './DomainTable.module.scss';
 //- Asset Imports
 import grid from './assets/grid.svg';
 import list from './assets/list.svg';
+import settings from './assets/settings.png';
```

Comment:
Can we use SVG instead of png?

Reply:
Huuh, it was replaced with svg @EthFish gave me, but looks like it is lost when I rebase. It is tooo old PR

Reply:
Keeping this png for now as we are unable to locate svg file